litmus-portal/backend/auth/go.sum

```diff
@@ -4,6 +4,7 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
Can you remove these changes

Reply:
My origin has been updated using the update PR button....now if I do a reset and remove the changes, then the changes that were added on top of my two commits will also we removed